sous-chef/sous_chef/grocery_list/generate_grocery_list.py

```diff
@@ -100,30 +103,26 @@ def upload_grocery_list_to_todoist(self, todoist_helper: TodoistHelper):
                     priority=2 if entry["get_on_second_shopping_day"] else 1,
                 )
 
-    def send_bean_preparation_to_todoist(self, todoist_helper: TodoistHelper):
+    def send_preparation_to_todoist(self, todoist_helper: TodoistHelper):
         # TODO separate service? need freezer check for defrosts
-        # TODO generalize beyond beans
-
-        bean_prep = self.config.bean_prep
-        project_name = bean_prep.project_name
+        project_name = self.config.project_name
         if self.config.todoist.remove_existing_task:
             [
                 todoist_helper.delete_all_items_in_project(project_name)
                 for _ in range(3)
             ]
 
-        due_date = self.due_date_formatter.get_due_datetime_with_hour_minute(
-            weekday=bean_prep.prep_day, hour=bean_prep.prep_hour, minute=0
-        )
-
-        # TODO change queue_bean_preparation into dataframe to use pandas here
-        if self.queue_bean_preparation:
-            for item in self.queue_bean_preparation:
-                label_list = item["group"].from_recipe + item["group"].from_day
+        if self.queue_preparation:
+            for _, row in self.queue_preparation.iterrows():
```

Comment:
Should add tests for adding to queue_preparation here

Reply:
added, please check